source/core/Math.cpp

```diff
@@ -278,6 +276,16 @@ using namespace std;
 		return _NextInstruction();													\
 		}
 
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES(TYPE) \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <  0) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >= 0) ) \
+    DECLARE_VIREO_COMPARISON_PRIMITIVES_BASE(TYPE)
+
+#define DECLARE_VIREO_COMPARISON_PRIMITIVES_BOOLEAN() \
+    DECLARE_VIREO_PRIMITIVE2( IsLT0Boolean, Boolean, Boolean, (_Param(1) = false) ) \
+    DECLARE_VIREO_PRIMITIVE2( IsGE0Boolean, Boolean, Boolean, (_Param(1) = true) ) \
```

Comment:
He meant '='.
I don't have strong opinions on your second comment.  Vireo is not strictly G, and it's not strictly important if it allows things G wouldn't, provided DFIR filters such cases out before Vireo ever sees them when the source is a G VI.